lib/galaxy/tools/xsd/galaxy.xsd

```diff
@@ -1218,9 +1218,7 @@ it may be inconvenient to upload the entiry file and this can be used instead.
     <xs:attribute name="decompress" type="PermissiveBoolean">
       <xs:annotation>
         <xs:documentation xml:lang="en"><![CDATA[
-
-If ``compare`` is set to ``diff``, attempt to decompress both produced output and expected output files if needed before performing the diff. This flag is useful for testing gzipped outputs that are non-deterministic despite having deterministic decompressed contents. This is available in Galaxy since release 17.05 and was introduced in [pull request #3550](https://github.com/galaxyproject/galaxy/pull/3550).
-
+This flag is useful for testing gzipped outputs that are non-deterministic despite having deterministic decompressed contents. Gzipped files (local and produced) are attempted to be decompressed if needed if ``compare`` is set to ``diff``. The default value is False. This is available in Galaxy since release 17.05 and was introduced in [pull request #3550](https://github.com/galaxyproject/galaxy/pull/3550).
```

Comment:
```suggestion
This flag is useful for testing compressed outputs that are non-deterministic despite having deterministic decompressed contents. Compressed files are attempted to be decompressed if needed if ``compare`` is set to ``diff``. The default value is False. This is available in Galaxy since release 17.05 and was introduced in [pull request #3550](https://github.com/galaxyproject/galaxy/pull/3550).
```